packages/components/grid/src/stories/pagination.stories.ts

```diff
@@ -64,40 +52,36 @@ export const Basic: Story = {
           margin-block: 1rem;
           justify-content: space-between;
         }
-
         sl-paginator {
           flex: 1;
         }
       </style>
-      <sl-grid .items=${people2.slice(startIndex, endIndex)}>
+      <sl-grid .items=${people2.slice(0, 10)}>
         <sl-grid-column path="firstName"></sl-grid-column>
         <sl-grid-column path="lastName"></sl-grid-column>
         <sl-grid-column path="profession"></sl-grid-column>
         <sl-grid-column path="status"></sl-grid-column>
         <sl-grid-column path="membership"></sl-grid-column>
       </sl-grid>
       <div class="pagination">
-        <sl-paginator-status .totalItems=${people2.length} .page=${page} .pageSize=${pageSize}></sl-paginator-status>
-        <sl-paginator
-          .totalItems=${people2.length}
-          .pageSizes=${pageSizes}
-          .page=${page}
-          .pageSize=${pageSize}
-        ></sl-paginator>
-        <sl-paginator-size .pageSizes=${pageSizes} .pageSize=${pageSize}></sl-paginator-size>
+        <sl-paginator-status .totalItems=${people2.length}></sl-paginator-status>
+        <sl-paginator @sl-page-change=${onPageChange} .totalItems=${people2.length}></sl-paginator>
```

Comment:
When I change the page size, the first page is not being set.

Reply:
Fixed